packages/compass-collection/src/components/collection/collection.tsx

```diff
@@ -2,21 +2,36 @@ import type AppRegistry from 'hadron-app-registry';
 import { createLoggerAndTelemetry } from '@mongodb-js/compass-logging';
 import type { Document } from 'mongodb';
 import React, { useCallback, useEffect } from 'react';
-import { TabNavBar, css } from '@mongodb-js/compass-components';
+import {
+  TabNavBar,
+  css,
+  withTheme,
+  uiColors,
+} from '@mongodb-js/compass-components';
 
 import CollectionHeader from '../collection-header';
+import type { StatsObject } from '../../modules/stats';
 
 const { track } = createLoggerAndTelemetry('COMPASS-COLLECTION-UI');
 
 function trackingIdForTabName(name: string) {
   return name.toLowerCase().replace(/ /g, '_');
 }
 
-const collectionStyles = css({
+const collectionLightStyles = css({
   display: 'flex',
   alignItems: 'stretch',
   height: '100%',
   width: '100%',
+  background: uiColors.white,
+});
+
+const collectionDarkStyles = css({
```

Comment:
Same as above, wouldn't it make sense to have only styles that change extracted? This should make maintaining this a bit easier over time